src/styles/components/_feedback.scss

```diff
@@ -0,0 +1,17 @@
+@import '../settings/colors';
+
+#w-feedback__dropdown {
+  display: none;
+}
+
+#w-feedback__dropdown + .w-feedback__label + .w-feedback__form {
```

Comment:
I think if you're using the existing summary/details element (https://web.dev/handbook/web-dev-components/#details) you may not need to do this.

Reply:
Don't think the web-dev-component will help if its being returned in the `render` function.